src/modals/Transfer.tsx

```diff
@@ -184,152 +201,130 @@ function Transfer(props: TransferProps) {
             }
             setEstimateLoading(false);
         }
-    }
+    };
 
-    async function handleTransferSmartWalletButtonClick() {
+    const handleTransferSmartWalletButtonClick = async () => {
         if (transfer.check) {
             await sendRBTC();
         } else {
             await transferSmartWalletButtonClick();
         }
-    }
+    };
+
+    const returnLoading = (loading: boolean) => (
```

Comment:
I think this could be a stateless component

Reply:
This has a state, since it is the loading icon that appears next to the button at the moment of pressing it. Since I reused the same code, It was necessary to add the state to identify which button was pressed.

Reply:
let me read it and do the necessary changes.